extensions/tekton/generator/cmd/generate/generate-v1beta1.go

```diff
@@ -30,6 +30,7 @@ func main() {
 	// no other types need to be defined as they are auto discovered
 	crdLists := map[reflect.Type]schemagen.CrdScope{
 		// v1beta1
```

Comment:
I also see a new resource `ResolutionRequest` in `resolution.tekton.dev/v1beta1` apiGroup. Do you think we should add it?

Reply:
I'll have a look at that